fail2ban/version.py

```diff
@@ -25,3 +25,5 @@
 __license__ = "GPL-v2+"
 
 version = "0.9.3.dev"
+
+agent_string = "Fail2Ban %s" % version
```

Comment:
Perhaps we can use "Fail2Ban/%s" to confirm [RFC 7231 5.5.3](https://tools.ietf.org/html/rfc7231#section-5.5.3)?